pkg/pillar/evetpm/tpm.go

```diff
@@ -773,3 +800,88 @@ func pcrBankSHA256EnabledHelper() bool {
 	_, err = tpm2.ReadPCR(rw, 0, tpm2.AlgSHA256)
 	return err == nil
 }
+
+func saveDiskKeySealingPCRs(pcrsFile string) error {
+	trw, err := tpm2.OpenTPM(TpmDevicePath)
+	if err != nil {
+		return err
+	}
+	defer trw.Close()
+
+	readPCRs, err := readDiskKeySealingPCRs()
+	if err != nil {
+		return err
+	}
+
+	frw, err := os.Create(pcrsFile)
```

Comment:
You should probably use WriteRename() for this file in /persist/ to avoid getting a zero length file if there is a sudden poweroff or crash.

Reply:
done.

Reply:
builds failed, reverted the last commit, still failing for some reason.